test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+tenacity==8.0.1
```

Comment:
I believe this file can be removed as we have the test requirements in `tox.ini`.